include/iso_alloc.h

```diff
@@ -7,6 +7,10 @@
 #include <stdio.h>
 #include <stdlib.h>
 
+#if !__aarch64__ && !__x86_64__
+#pragma message "IsoAlloc is untested and unsupported on 32 bit platforms"
```

Comment:
Unsupported, or it will simply not work?
Also, maybe we should do something like `assert(sizeof(size_t) >= 64`)` instead?

Reply:
Yah 32 bit won't work, should be more explicit about that.